importer.py

```diff
@@ -40,11 +41,11 @@ def get_params(self, path):
                     lng = float(match.group('lng'))
                 except: pass
         if not camera:
-            print 'did not detect camera name, assuming "default"'
+            logging.warn('did not detect camera name, assuming "default"')
```

Comment:
If this is a name placeholder, how about "Unnamed" instead so convey that it's not related to a system default.